testsuite/features/step_definitions/salt_steps.rb

```diff
@@ -11,7 +11,7 @@
   # 300 is the default 1st keepalive interval for the minion
   # where it realizes the connection is stuck
   repeat_until_timeout(timeout: 300, retries: 3, message: "Master can not communicate with #{minion}", report_result: true) do
-    out, _code = $server.run("salt #{system_name} test.ping")
+    out, _code = $server.run("salt #{system_name} test.ping", check_errors: false, timeout: 90)
```

Comment:
when one increases a timeout, one should always ask himself/herself the question:

"would a customer find this timeout acceptable?"

Is 90 seconds something acceptable, or rather the sign of a problem of our product?

Reply:
A ping can't definitely not take 90 seconds.
Instead we should just have a 2-3 seconds timeout, and re-try the ping X seconds until it replies.

Reply:
When the minion is down, then the ping can take up to 300s as noted in the comment above.
Timeout 90 ensures that it is tried 3 times within this 300s.

Reply:
Let's try it!